verification/src/scheduler.rs

```diff
@@ -0,0 +1,61 @@
+use cron::Schedule;
+use futures::Future;
+
+use chrono::Utc;
+
+pub fn spawn_job_with_schedule<F, Fut>(
+    schedule: Schedule,
+    mut run: F,
+) -> tokio::task::JoinHandle<()>
+where
+    F: (FnMut() -> Fut) + Send + 'static,
+    Fut: Future + Send + 'static,
+    <Fut as futures::Future>::Output: Send,
+{
+    tokio::spawn(async move {
+        loop {
+            let sleep_duration = time_till_next_call(&schedule);
+            log::debug!("sleep for {:?}", sleep_duration);
```

Comment:
`scheduled next compiler list update in {:?}`

Reply:
this function doesn't know about compiler list. I thought about passing name of job as function argument (&str) and print it in logs, what do you think?

Reply:
> I thought about passing name of job as function argument (&str) and print it in logs, what do you think?

I believe it may be a good idea